src/app/contexts/EventContext/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import { node } from 'prop-types';
+import { useWindowEvent } from './useWindowEvent';
+import { useHandlerMap } from './useHandlerMap';
+
+export const EventContext = React.createContext({
+  useWindowEvent,
+  useClickTracker: () => {},
+});
+
+export const EventContextProvider = ({ children }) => {
+  const [handlerMap, setHandlerMap] = useState({});
+
+  const useClickTracker = (attr, handler) => {
```

Comment:
As far as I can see useClickTracker doesnt have any tests? I might be missing some, but it has some very hard to read logic we should def fully unit test